setup.py

```diff
@@ -39,5 +40,5 @@ def get_version():
         description="Help with running stuff",
         long_description="View the github page " +
             "(https://github.com/EDITD/runners) for more details.",
-        install_requires=REQUIREMENTS
+        install_requires=REQUIREMENTS,
```

Comment:
does this still work with python2? I'm now confused because of https://github.com/EDITD/docs/pull/396/commits/d8c13d9316e36f192f6e7827d7001d90489f8927

Reply:
Yep it's fine, it's only trailing commas after `**kwargs` being passed into a function that doesn't pass python 2 linting.